packages/app-project/src/screens/ProjectHomePage/components/AboutProject/AboutProject.mock.js

```diff
@@ -0,0 +1,6 @@
+export const AboutProjectMock = {
+  project: {
+    display_name: 'Test Project Name',
+    introduction: 'Test Project Name has this introduction...'
+  }
```

Comment:
[medium suggestion] In most scenarios, I'd recommend making sure that .display_name is significantly different from .introduction.

e.g. `display_name: "Whale Song Research", introduction: "We need to figure out what these damn cetaceans are yakking on about."`

The reason: if we're just using "Test Project Name" and "Test Project Name has an introduction", then if the _display name fails to render,_ but the _introduction paragraph renders properly,_ then the test "does project.display_name render?" will give a false positive, because the text "Test Project Name" exists in the introduction text.